### ISSUE TITLE
Vizier query_constraints returns "Empty TableList"

### ISSUE THREAD

**Issue by hippke** [astropy/astroquery#1326]
I try to pull a table from Vizier with some constraints, but get an "Empty TableList" result:
`from astroquery.vizier import Vizier`
`result = Vizier(columns=["*"], catalog="IV/34/epic").query_constraints(ID="247887989")`
`print(result)`

**Comment by keflavich** [comment]
This is a bug.  As a workaround, do:
```
result = Vizier(columns=["*"]).query_constraints(ID="247887989", catalog="IV/34/epic")
```
but I'll have a fix out shortly.

**Comment by hippke** [comment]
Thanks! Workaround works for me.

**Comment by bsipocz** [comment]
Fixed by #1328
